pbiviz.json

```diff
@@ -4,7 +4,7 @@
     "displayName": "WordCloud",
     "guid": "WordCloud1447959067750",
     "visualClassName": "WordCloud",
-    "version": "1.2.14",
+    "version": "1.2.15",
```

Comment:
It'd be better to use `1.3.0` because of fact that a new feature has been implemented.
Let's follow semver